### ISSUE TITLE
Improved SWC Sign Up Flow

### ISSUE THREAD

**Issue by coinbasemar** [Stand-With-Crypto/swc-web#727]
We should update our sign up flow with the following:

- Phone number, first name, last name, address is required to sign up
- Show the "Join" GIF as an incentive to sign up
- Show suggested next actions after signing up

Designs: https://www.figma.com/file/r9ilZQ4QMRpjefdQFq93Az/SWC-SMS?type=design&node-id=1-2947&mode=design

**Comment by travisbloom-cb** [comment]
eng notes: 
- this is designed with full page modals, which is out of scope for this ticket
- We need to defer giving people an NFT for creating an account until they fill in their profile info
- We should require all the profile info to submit the form now

**Comment by travisbloom-cb** [comment]
This is blocked by #725

**Comment by travisbloom-cb** [comment]
We should also structure this as an A/B test, with a control and variant for the new flow. See [experiments](https://github.com/Stand-With-Crypto/swc-web/blob/639-show-whether-a-position-is-for-or-against-crypto/src/utils/shared/experiments.ts#L29)
